### DISCUSSION TITLE
Feature request: virtual scroll for select-menu

### DISCUSSION THREAD

**Discussion by VincentQDoLM** [RocketCommunicationsInc/astro#1006] [Feature Requests]
Select menu have performance issue when more than 1000 items are in the list. Users requesting virtual scroll for the select menu for performance improvement. https://astro-components.netlify.app/?path=/docs/forms-select-menu--select-menu

**Comment by micahjones13** [comment]
Hey there, thanks for raising this! 

The use of select with that many items is not recommended - see our compliance guidelines [here](https://www.astrouxds.com/components/select/). Because this goes against our compliance, we won't be implementing virtual scroll for rux-select at this time. However, if we rewrite our rux-select in the future, we will definitely take this into consideration.
